packages/web-react/src/components/FileUploader/README.md

```diff
@@ -337,6 +339,8 @@ The rest of the properties are created from the default `<input>` element. [More
 | `label`               | `string`        | -       | ✕        | Label for the list                            |
 | `UNSAFE_className`    | `string`        | -       | ✕        | FileUploaderList custom class name            |
 | `UNSAFE_style`        | `CSSProperties` | -       | ✕        | FileUploaderList custom style                 |
+| `hasImagePreview`     | `boolean`       | false   | ✕        | Show image preview in the list                |
+| `isEditable`          | `boolean`       | false   | ✕        | Show edit button for image files in list      |
```

Comment:
Please sort the new props alphabetically and before the UNSAFE props (we keep them always last).

Reply:
sorted